store/tikv/client_batch.go

```diff
@@ -67,6 +64,8 @@ func (a *batchConn) fetchAllPendingRequests(
 	maxBatchSize int,
 	entries *[]*batchCommandsEntry,
 	requests *[]*tikvpb.BatchCommandsRequest_Request,
+	addr string,
```

Comment:
It's better to add a `target` field in `batchConn`.

Reply:
Done